workers/unity/Packages/com.improbable.gdk.core/CodegenAdapters/ComponentDispatcherHandler.cs

```diff
@@ -60,5 +59,19 @@ protected Entity TryGetEntityFromEntityId(EntityId entityId)
 
             return entity;
         }
+
+        /// <summary>
+        ///     An exception for when the ComponentDispatcherHandler receive a CommandRequest or CommandResponse
+        ///     with an index that it does not know.
+        /// </summary>
+        protected class UnknownCommandIndexException : Exception
+        {
+            /// <param name="commandIndex">The unknown command index.</param>
+            /// <param name="componentName">The name of the component that the command was received on.</param>
+            public UnknownCommandIndexException(uint commandIndex, string componentName)
+                : base($"Unknown command index: {commandIndex} received for {componentName}")
```

Comment:
Is there any way a user can cause this that doesn't involve magic? If not then we might need to be clearer in the error message.

Reply:
Actually this index is generated by the connection, not by the bridge, so we know any commands we receive responses for must have been sent by this worker.